packaging/scripts/preinstall.sh

```diff
@@ -101,4 +101,9 @@ if [ "$1" = "upgrade" ]; then
     copyDbToUserDir
 fi
 
+# copy current binary to /tmp/oldevcc (for possibility to rollback)
+  if [ -f /usr/bin/evcc ]; then
```

Comment:
I'd use the `"$1" = upgrade` check here, just to stay within the overall "standards" this script tries to apply.
Simply moving the cp into the block above below copyDbToUserDir should do the trick.

Reply:
That's right. That should work. Thank you. I will apply it.